web/html/src/manager/virtualization/guests/properties/guest-disks-panel.js

```diff
@@ -200,36 +165,56 @@ function guestDiskFields(model: Object, index: number,
 }
 
 function guestDisksPanel(model: Object, changeModel: Function, pools: Array<Object>, caps: Object): React.Node {
```

Comment:
Just curious: why is this not implemented as a component?

Reply:
That is coming from before the Form context. IIRC that broke the Form inputs, but yeah at some point in time it would be nice to convert into a component